com.unity.render-pipelines.high-definition/Editor/VFXGraph/Outputs/VFXLitCubeOutput.cs

```diff
@@ -52,8 +52,20 @@ protected override IEnumerable<string> filteredOutSettings
                 foreach (var setting in base.filteredOutSettings)
                     yield return setting;
 
-                yield return "blendMode";
-                yield return "shaderGraph";
+                yield return nameof(blendMode);
+                yield return nameof(shaderGraph);
+            }
+        }
+
+        protected override IEnumerable<string> untransferableSettings
+        {
+            get
+            {
+                foreach (var setting in base.untransferableSettings)
+                {
+                    yield return setting;
+                }
+                yield return nameof(shaderGraph);
```

Comment:
This change is to fix a different issue, keeping exposed property even after converting an output to a type of output that does not support shadergraph.